pom.xml

```diff
@@ -64,16 +64,7 @@
     <github.global.server>github</github.global.server>
     <site.installationModule>google-cloud-bigquery-parent</site.installationModule>
     <google.core.version>1.93.4</google.core.version>
```

Comment:
Looks like you can remove this one too

Reply:
unforch no due to https://github.com/googleapis/java-bigquery/blob/b24d6354196724195aa810fe50c6a006f04b41c7/google-cloud-bigquery/pom.xml#L65